synapse/handlers/room.py

```diff
@@ -89,6 +70,54 @@ def __init__(self, hs):
         self.room_member_handler = hs.get_room_member_handler()
         self.config = hs.config
 
+        self._presets_dict = {
+            RoomCreationPreset.PRIVATE_CHAT: {
+                "join_rules": JoinRules.INVITE,
+                "history_visibility": "shared",
+                "original_invitees_have_ops": False,
+                "guest_can_join": True,
+                "power_level_content_override": {"invite": 0},
+                "encryption_algorithm": (
```

Comment:
why not just a boolean `encryption_enabled` here and substitute `RoomEncryptionAlgorithms.DEFAULT` later on? seems simpler/less repetitive.